betka.sh

```diff
@@ -5,7 +5,9 @@
 # The result directory will contain generated sources from upstream directories
 #
 
-set -ex
+set -eEx
```

Comment:
-x is not needed.

Reply:
I can remove it. But the -x was present before this PR too.

Reply:
It is removed in [12dc547](https://github.com/sclorg/container-common-scripts/pull/341/commits/12dc547d1bf8623b5835e16d323bd3ac36ca0724).